style.css

```diff
@@ -1 +1,217 @@
 /*  Add your styling here */
+
+#navigation{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    position: fixed;
+    align-items: center;
+    top: 0px;
+    background-color: white;
+    overflow: hidden;
+    height: 60px;
+}
+#logo{
+    margin-top: 0px;
+    padding-top: 0px;
+
+    height: 50px;
+    width: 120px;
+}
+a{
```

Comment:
what happens when you hover over hyperlinks? consider using a pseudo class to style `a` html elements as well